### PR TITLE
Enhancement clubpage functionality

### PR DESCRIPTION
**Pull request by alevtyna30**

### CODE REVIEW

File: src/main/java/com/academy/ui/components/CommentsClubComponent.java [hunk 1]
```diff
@@ -8,9 +8,9 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+@Getter
 public class CommentsClubComponent extends BaseComponent {
 
-    @Getter
     @FindBy(xpath = "//button[contains(@class,'comment-button')]")
     private WebElement leaveCommentButton;
```
Comment (lhalam) [inline src/main/java/com/academy/ui/components/CommentsClubComponent.java]: move to Clubpage

File: src/main/java/com/academy/ui/components/CommentsClubComponent.java
Comment (lhalam) [inline src/main/java/com/academy/ui/components/CommentsClubComponent.java:1]: image

File: src/main/java/com/academy/ui/components/RatingComponent.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+@Getter
+public class RatingComponent extends BaseComponent {
+
+    @FindBy(xpath = "//ul[@role='radiogroup']")
```
Comment (lhalam) [inline src/main/java/com/academy/ui/components/RatingComponent.java]: `.`

**Review by nasock — APPROVED** [review]

File: src/main/java/com/academy/ui/components/AddChildPopUpComponent.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+@Getter
+public class AddChildPopUpComponent extends BasePopUp {
+
+    @FindBy(xpath = "//input[@id='add-child_firstName']")
```
Comment (lhalam) [inline src/main/java/com/academy/ui/components/AddChildPopUpComponent.java]: `//->./`

Comment (alevtyna30) [reply src/main/java/com/academy/ui/components/AddChildPopUpComponent.java]: upd all class according to comment

**Pull request merged**